dspace/config/dspace.cfg

```diff
@@ -780,11 +780,11 @@ event.dispatcher.default.class = org.dspace.event.BasicDispatcher
 # Add rdf here, if you are using dspace-rdf to export your repository content as RDF.
 # Add iiif here, if you are using dspace-iiif.
 # Add orcidqueue here, if the integration with ORCID is configured and wish to enable the synchronization queue functionality
-event.dispatcher.default.consumers = versioning, discovery, eperson
+event.dispatcher.default.consumers = versioning, discovery, eperson, submissionconfig
 
 # The noindex dispatcher will not create search or browse indexes (useful for batch item imports)
 event.dispatcher.noindex.class = org.dspace.event.BasicDispatcher
-event.dispatcher.noindex.consumers = eperson
+event.dispatcher.noindex.consumers = eperson, submissionconfig
```

Comment:
Why are we added this new consumer to the `noindex` dispatcher?  If the `submissionconfig` needs to trigger reindexing of the Collection, then it shouldn't be listed here as it's writing to the index.

Reply:
Sorry, I've made several tests, including changing this configuration to understand why the collection wasn't available. I will undo this change that doesn't affect how it works.